public-ui/src/app/views/ehr-view/ehr-view.component.ts

```diff
@@ -195,5 +190,11 @@ export class EhrViewComponent implements OnInit, OnDestroy {
       this.showGenderIdentity = false;
     }
   }
-
+  public toggleSynonyms(conceptId) {
+    if (!this.showMoreSynonyms[conceptId]) {
```

Comment:
nit: would probably just do a `this.showMore[..] = !this.showMore[..]`

Reply:
Done.